docs/src/quickstart/index.md

```diff
@@ -0,0 +1,223 @@
+
+[Windows](##ref-qs-win)
+[Linux](##ref-qs-linux)
+[MacOS](##ref-qs-mac)
+
+# {#ref-qs-win} Windows
+
+Reach requires [make](https://en.wikipedia.org/wiki/Make_(software)), [Docker](https://www.docker.com/get-started), and [Docker Compose](https://docs.docker.com/compose/install/).
+
+## {#ref-qs-win-prereqs} Prerequisites
+
+* Windows 10 installed with version 2004 or higher
+
+## {#ref-qs-win-install} Installation
+
+Click the Windows icon, type `CMD`, and then click `Run as Administrator`.
+There are a number of commands that need to be run to get Windows ready for Reach.
+
+In the Command Prompt window, run:
+
+``` cmd
+$ wsl --install -d ubuntu
+```
+
+This installs Ubuntu WSL, which is used via command line later.
+
+In the Command Prompt window, run:
+
+``` cmd
+$ wsl --set-version ubuntu 2
+``` 
+
+This sets the WSL to version 2, which is required by Docker and Reach.
+
+Next, download and install [Docker Desktop](https://www.docker.com/products/docker-desktop).
+
+:::note
+Make sure to follow Docker's instructions for installation and configuration.
+Those instructions are subject to change and are outside of our control.
+:::
+
+Click the `Settings` (gear) icon along the top of the Docker app.
+
+![Settings icon in Docker`](/quickstart/settings-icon.png)
+
+Verify that `WSL 2.0` is checked, as well as `Use Docker Compose V2`.
+
+![WSL 2 settings in Docker`](/quickstart/wsl-2.png)
+![Docker Compose in Docker`](/quickstart/docker-compose.png)
+
+Click `Resources` in the left-hand menu, and make sure that that `Enable integration with additional distros` is checked, and that `Ubuntu` is selected.
+
+![WSL integration in Docker`](/quickstart/wsl-integrate.png)
+
+Click the Ubuntu icon in the Windows Start-up menu to open the Ubuntu terminal.
+You will need to provide a `username` and `password` for Ubuntu.
+
+In the terminal, run the following to install `make`:
+
+```cmd
+$ sudo apt install make
+```
+
+Next, run the following to allow `apt` to use repositories containing https:
+
+```cmd
+$ sudo apt-get install ca-certificates curl gnupg lsb-release
+```
+
+Create and navigate to the reach directory with the following command:
+
+``` cmd
+$ mkdir -p ~/reach && cd ~/reach
+```
+
+Download Reach with the following command:
+
+``` cmd
+$ curl https://docs.reach.sh/reach -o reach ; chmod +x reach
+```
+
+Reach is successfully downloaded if the following command returns a version number:
+
+```
+$ ./reach version
+```
+
+You are now ready to start programming in Reach.
+Check out our [Rock Paper Scissors!](##tut) tutorial to get started.
+
+# {#ref-qs-linux} Linux
+
+Reach requires [make](https://en.wikipedia.org/wiki/Make_(software)), [Docker Engine](https://docs.docker.com/get-docker/), and [Docker Compose](https://docs.docker.com/compose/install/).
+
+## {#ref-qs-linux-prereqs} Prerequisites
+
+* A compatible version of Linux.
+Check the [Docker Engine](https://docs.docker.com/engine/install/) page for supported distros. 
```

Comment:
Do you want this indented to be in the same list item?

Reply:
I intended it to be the next sentence of the list item, but it could look better if it is a paragraph below and indented the same.